modules/ROOT/partials/configuration/inline-css.adoc

```diff
@@ -0,0 +1,30 @@
+=== inline_selector_filter
+
+Determines whether it is valid for a given CSS selector to have its CSS properties inlined into the HTML content
+
+Default: All selectors are considered valid to have their CSS inlined
+
+Type: `String` `RegExp` or `Function`
+
+[source,js]
+----
+inlinecss_selector_filter: (selector: string): boolean => {
+	return selector.indexOf('myprefix') !== -1;
+}
+----
```

Comment:
```suggestion
[source,ts]
----
inlinecss_selector_filter: (selector: string): boolean => {
  return selector.indexOf('myprefix') !== -1;
}
----
```

Reply:
Changing `js` to `ts` for Typescript here to support type declarations because they are only supported in Typescript (sugar syntax for Javascript)